.jshintrc

```diff
@@ -2,5 +2,7 @@
   "esnext": true,        // ES6 syntax support
   "globals": {
     "ko": true
-  }
+  },
+  "node": true,
+  "mocha": true
```

Comment:
Similarly, we are using mocha, a unit testing framework. So we tell node that it should not complain when we use globals like `describe(...)`, `beforeEach(...)` and `it(...)`